packages/renderer/src/get-compositions.ts

```diff
@@ -1,10 +1,16 @@
-import puppeteer from 'puppeteer';
+import puppeteer from 'puppeteer-core';
 import {TCompMetadata} from 'remotion';
+import {getLocalChromiumExecutable} from './get-local-chromium-executable';
 
 export const getCompositions = async (
 	webpackBundle: string
 ): Promise<TCompMetadata[]> => {
-	const browser = await puppeteer.launch();
+	const executablePath = await getLocalChromiumExecutable();
+	console.log(executablePath);
+	const browser = await puppeteer.launch({
+		executablePath,
+		args: ['--no-sandbox', '--disable-setuid-sandbox'],
```

Comment:
Why do we need no-sandbox now?

Reply:
We don't need it now. But the puppeteer instance was not launching and I went to puppeteer troubleshooting docs for linux and it said to provide `no-sandbox` arg. That's why I included it.